ebean-test/src/test/java/org/tests/batchinsert/TestBatchInsertFlush.java

```diff
@@ -42,19 +42,22 @@ public void batchFlush() {
 
       for (int i = 0; i < 3; i++) {
         Customer customer = ResetBasicData.createCustomer("BatchFlushPreInsert " + i, null, null, 3);
-        customer.addContact(new Contact("Fred" + i, "Blue"));
+        customer.addContact(new Contact("BatchFlush" + i, "Blue"));
         customers.add(customer);
       }
 
       for (int i = 3; i < 6; i++) {
         Customer customer = ResetBasicData.createCustomer("BatchFlushPostInsert " + i, null, null, 3);
-        customer.addContact(new Contact("Fred" + i, "Blue"));
+        customer.addContact(new Contact("BatchFlush" + i, "Blue"));
```

Comment:
I renamed it, because ResetBasicData.reset() creates also contacts with `Fred` (So the delete in `finally` is easier